docs/content/operator/common/network.mdx

```diff
@@ -0,0 +1,28 @@
+import TestnetNodeConfig from "../../_snippets/network-node-config-testnet.mdx";
+import DevnetNodeConfig from "../../_snippets/network-node-config-devnet.mdx";
+
+---
+title: Network
+description: A network configuration, peering setup for IOTA networks.
+tags: [node-operation]
+teams:
+  - iotaledger/node
+---
+
+To synchronize the state, node needs to establish connections with other nodes that have already joined the network.
+
```

Comment:
Here we should probably explain again to open up the `8084` port and setup an external address for discovery.

Reply:
Fixed in https://github.com/iotaledger/iota/pull/5161/commits/f0ff0909284521d1b20cf1429dca99888988612d .